source/dataman/CacheMan.h

```diff
@@ -41,7 +41,7 @@ class CacheItem
                                             const std::vector<size_t> &o)
     {
         std::vector<size_t> g;
-        for (int i = 0; i < p.size(); i++)
+        for (int i = 0; i < p.size(); ++i)
```

Comment:
@chuckatkins This certainly looks more clean. I am just curious how the efficiency compares between a for loop and std::transform + lambda function. I know this case is trivial, but which should I use for large functions?